query/common/src/test/groovy/org/codice/ddf/admin/common/fields/base/BaseObjectFieldTest.groovy

```diff
@@ -118,4 +118,25 @@ class BaseObjectFieldTest extends Specification {
         then:
         subFieldOfInnerField.getValue() == 'valueChange'
     }
+
```

Comment:
It's a pretty rare edge case, especially since the size is being checked. I personally think it's a bit overkill. Duplicate error codes also doesn't break anything.